main.go

```diff
@@ -159,8 +159,8 @@ func (c *Controller) handleErr(err error, key interface{}) {
 		return
 	}
 
-	// This controller retries 5 times if something goes wrong. After that, it stops trying.
-	if c.queue.NumRequeues(key) < 5 {
+	// This controller retries 30 times if something goes wrong. After that, it stops trying.
+	if c.queue.NumRequeues(key) < 30 {
```

Comment:
The description mentions: "10 retries result in about a 4 second window". Is the description out of date? Or should I assume 3x (i.e., a 12 second window)?

Reply:
Since the queue uses an exponential back-off, I'm not really sure what the actual duration for 30 retries will be. I've mentioned the 10 retries as a value I've measured empirically.

I've set 30 as it should be well within anything I've observed, but given how fickle the BZ is to reproduce, I wanted to err on the side of generosity unless there's a good reason otherwise.

Happy to try something else if you've think this isn't the right thing to do this.